### PR TITLE
Add back skeleton for PR

### PR DESCRIPTION
**Pull request by kerriannercrawford**
- Skeleton created for component

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -0,0 +1,44 @@
...
+  ],
+  "author": "Blended Edge",
+  "license": "Apache-2.0",
+  "bugs": {
+    "url": "https://github.com/blendededge/soap-component-oih/issues"
+  },
+  "homepage": "https://github.com/blendededge/soap-component-oih#readme",
+  "dependencies": {
+    "@openintegrationhub/ferryman": "^2.1.0",
+    "axios": "^0.25.0",
```
Comment (winklerj) [inline package.json]: Is there a reason to include Axios?

File: package.json [hunk 2]
```diff
@@ -0,0 +1,44 @@
...
+    "@types/node": "^17.0.10",
+    "@typescript-eslint/eslint-plugin": "^5.10.0",
+    "@typescript-eslint/parser": "^5.10.0",
+    "chai": "^4.3.4",
+    "eslint": "^8.7.0",
+    "mocha": "^9.2.0",
+    "nock": "^13.2.2",
+    "testdouble": "^3.16.4",
+    "typescript": "^4.5.5",
+    "yarn": "^1.22.17"
```
Comment (winklerj) [inline package.json]: I would like to standardize on npm since the latest features are on par with yarn

File: Dockerfile [hunk 3]
```diff
@@ -0,0 +1,19 @@
...
+COPY package.json /usr/src/app
+
+RUN yarn install --production
+
+COPY . /usr/src/app
+
+RUN chown -R node:node .
+
+USER node
+ENTRYPOINT ["node", "./node_modules/@openintegrationhub/ferryman/runGlobal.js"]
```
Comment (winklerj) [inline Dockerfile]: Let's add the ferryman backwards compatible script to the repo checking for the new environment variable

File: tsconfig.tsbuildinfo [hunk 4]
```diff
@@ -0,0 +1 @@
...,[5,1],[29,1],[30,1],[31,1],[32,1],[6,1],[33,1],[34,1],[35,1],[36,1],[7,1],[41,1],[37,1],[38,1],[39,1],[40,1],[1,1],[42,1],[11,1],[10,1],[44,1]],"exportedModulesMap":[[45,1],[46,2],[47,2],[49,3],[50,4],[51,5],[52,6],[53,7],[54,8],[55,9],[56,10],[57,11],[58,12],[59,12],[60,13],[61,14],[62,15],[63,16],[48,1],[95,1],[64,17],[65,18],[66,19],[96,20],[67,21],[68,22],[69,23],[70,24],[71,25],[72,26],[73,27],[74,28],[75,29],[76,30],[77,31],[78,32],[80,33],[79,34],[81,35],[82,36],[83,37],[84,38],[85,39],[86,40],[87,41],[88,42],[89,43],[90,44],[91,45],[92,46],[93,47],[94,48],[8,1],[9,1],[13,1],[12,1],[2,1],[14,1],[15,1],[16,1],[17,1],[18,1],[19,1],[20,1],[21,1],[3,1],[4,1],[43,1],[25,1],[22,1],[23,1],[24,1],[26,1],[27,1],[28,1],[5,1],[29,1],[30,1],[31,1],[32,1],[6,1],[33,1],[34,1],[35,1],[36,1],[7,1],[41,1],[37,1],[38,1],[39,1],[40,1],[1,1],[42,1],[11,1],[10,1],[44,1]],"semanticDiagnosticsPerFile":[45,46,47,49,50,51,52,53,54,55,56,57,58,59,60,61,62,63,48,95,64,65,66,96,67,68,69,70,71,72,73,74,75,76,77,78,80,79,81,82,83,84,85,86,87,88,89,90,91,92,93,94,8,9,13,12,2,14,15,16,17,18,19,20,21,3,4,43,25,22,23,24,26,27,28,5,29,30,31,32,6,33,34,35,36,7,41,37,38,39,40,1,42,11,10,44]},"version":"4.5.5"}
```
Comment (winklerj) [inline tsconfig.tsbuildinfo]: Let's add this file to the .gitignore so we build from scratch on the build server

**Review by winklerj — CHANGES_REQUESTED** [review]

File: package.json (hunk 1 above)
Comment (kerriannercrawford) [reply package.json]: Nope, was taking dependencies from the REST component, forgot it wouldn't be necessary

**Review by winklerj — APPROVED** [review]

**Pull request merged**